action_server/src/sema4ai/action_server/_protocols.py

```diff
@@ -183,3 +191,59 @@ class ArgumentsNamespaceStart(ArgumentsNamespaceBaseImportOrStart):
     expose: bool
     expose_allow_reuse: bool
     api_key: str
+
+
+class ArgumentsNamespacePackageOrganizations(ArgumentsNamespace):
+    command: Literal["package"]
+    package_command: Literal["organizations"]
```

Comment:
Question: does it make sense that getting the organizations is something under `cloud`? Also, just with `organizations` it's a bit unclear whether it's a set or a get. 

Suggestion:

`action-server package list-organizations`

Reply:
It's true that there is no package related in this command. I'll move it to be subcommand of cloud command.